container/container_windows.go

```diff
@@ -101,7 +101,8 @@ func (container *Container) UpdateContainer(hostConfig *containertypes.HostConfi
 		resources.CPUCount != 0 ||
 		resources.CPUPercent != 0 ||
 		resources.IOMaximumIOps != 0 ||
-		resources.IOMaximumBandwidth != 0 {
+		resources.IOMaximumBandwidth != 0 ||
+		resources.NanoCPUs != 0 {
```

Comment:
nit: I updated the check here to include NanoCPUs in a different PR and forgot I asked you to add it to this one 👼. You can remove this check now. Sorry for the inconvenience.